universal-application-tool-0.0.1/app/models/Account.java

```diff
@@ -14,9 +14,22 @@
 import javax.persistence.Table;
 import services.program.ProgramDefinition;
 
+/**
+ * An EBean mapped class that represents a single user account in CiviForm. Users of all roles have
+ * an {@code Account}.
+ *
+ * <p>When a user logs in for the first time using SSO or fills out an application as a guest,
+ * CiviForm creates an {@code Account} record for them.
+ *
+ * <p>Note that residents have a single {@code Account} but may have multiple {@code Applicant}
+ * records associated with that account, one for each {@code Version} they have interacted with.
```

Comment:
Question (blocking): Would you clarify what this paragraph means? In particular, `one (Applicant) for each {@code Version} they have interacted with`. My understanding is that Applicant is also not versioned.

Reply:
My understanding is the same as @bion 's (one Applicant for each Version they've interacted with). But I could be wrong...

Reply:
Applicant is versioned because it contains ApplicantData which is tied to a specific version / question set.

Reply:
I don't believe that this is correct, yeah.  I believe there is in practice only ever one Applicant object.

Reply:
Updated, please TAL

Reply:
LGTM